File: frontend/src/app/validation/route-validation-status.ts
export class RouteValidationStatus {
    public network: String;
    public osmRouteId: Number;
    public line: string;
    public lineNumber: number;
    public name: String;
    public transportMode: String;
    public matching: String;
    public issue: String;
    
    public constructor(status: RouteValidationStatus) {
        this.network = status.network;
        this.osmRouteId = status.osmRouteId;
        this.line = status.line;
        this.lineNumber = status.lineNumber;
        this.name = status.name;
        this.transportMode = status.transportMode;
        this.matching = status.matching;
        this.issue = status.issue;
    }
}

File: src/main/java/nl/haltedata/validation/model/RouteValidationStatus.java
package nl.haltedata.validation.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
public class RouteValidationStatus {
    @Id
    private Long osmRouteId;
    private String network;
    private String line;
    private String name;
    private String transportMode;
    private String matching;
    private String issue;
}
